tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "rootDir": ".",
-    "outDir": "build"
+    "outDir": "build",
+    "target": "es2016"
```

Comment:
Is es2016 equivalent to Node 6?

Reply:
No :(  All the tests pass here pass, but I can't promise that we won't run into trouble down the road.

Reply:
Then why not be more conservative and use `es6` (aka `es2015`)?

Reply:
There's no direct relation between Node.js releases and a specific TypeScript target.  Here's my rejected feature request on the TypeScript repo:
https://github.com/Microsoft/TypeScript/issues/22306

For the moment, everything works with es2016.  I was optimizing for the most current feature set that passes all the tests.  The code that gets generated here is likely closer to the origin code, which makes for better debugging and likely better perf.

Reply:
I know that there's no direct relation between node and a TS target. But AFAIK Node 6 is closer to es2015 vs es2016. Having said that, it's probably fine because we have a set of tests with a pretty good coverage.

Reply:
Looking at https://node.green, as far as I can tell, the only thing missing for ES2016 for Node 6 is exponentiation operator. I think we're pretty safe to use es2016 as the target here.

Reply:
Great. Good to know!